jujugui/static/gui/src/app/app.js

```diff
@@ -754,7 +754,7 @@ YUI.add('juju-gui', function(Y) {
           const selectedModel = this._pickModel(modelList);
           if (selectedModel === null) {
             console.log('cannot select a model: using unconnected mode');
-            // XXX Drop the user into the unconnected state.
```

Comment:
We should probably have a comment here just saying that this drops the user into an unconnected mode.